tinyalloc.c

```diff
@@ -1,5 +1,6 @@
 #include "tinyalloc.h"
 #include <stdint.h>
+#include "os/os.h"
```

Comment:
Also add `#if MYNEWT_VAL(OS_SCHEDULING)` around this one?

Reply:
MYNEWT_VAL is unknown without os.h